### DISCUSSION TITLE
select from existing user names when creating a new account

### DISCUSSION THREAD

**Discussion by 117503445** [dani-garcia/vaultwarden#2927] [Ideas]
A person's username is often the same across sites. For new sites, consider selecting from a user's existing list of usernames to avoid duplicate typing

**Comment by stefan0xC** [comment]
This would be a feature for the Bitwarden clients and not Vaultwarden as the server side does not really have any control over how the clients themselves work.

Looking in the Bitwarden Community forum there once was an proposal for something like that: https://community.bitwarden.com/t/default-usernames/19071 (but the PR was not merged and not sure if someone is still working on this feature).
